modules/installation-configuration-parameters.adoc

```diff
@@ -788,6 +789,27 @@ ifdef::rhv[]
 |`platform.ovirt.ingress_vip`
 |Required. An IP address on the machine network that will be assigned to the Ingress virtual IP (VIP).
 |String. Example: `10.46.8.232`
+
+|`platform.ovirt.affinityGroups`
+|Required. A list of affinity groups that should be created during the installation process.
+|List of objects.
+
+|`platform.ovirt.affinityGroups.description`
+|Required. A description of the affinity group.
+|String. Example: `AffinityGroup for spreading each compute machine to a different host`
+
+|`platform.ovirt.affinityGroups.enforcing`
+|Required. When set to `true`, {rh-virtualization} does not provision any machines if not enough hardware nodes are available. When set to `false` {rh-virtualization} does provision machines even if not enough hardware nodes are available, resulting in multiple virtual machines being hosted on the same physical machine.
+
+|String. Example: `true`
+
+|`platform.ovirt.affinityGroups.name`
+|Required. A name for the affinity group.
```

Comment:
The name

Reply:
@stoobie ^^

Reply:
@Gal-Zaidman Changed to "The name of the affinity group."